Vic2ToHoI4/Source/HOI4World/HoI4World.cpp

```diff
@@ -1043,4 +1044,35 @@ std::set<std::string> HoI4::World::getSouthAsianCountries() const
 	}
 
 	return southAsianCountries;
+}
+
+void HoI4::World::buildConquerStrategies()
+{
+	for (auto& country: countries)
+	{
+		for (const auto& strategy: country.second->getAIStrategies())
+		{
+			if (strategy.getType() == "conquer_prov")
+			{
+				if (auto HoI4Tag =
+						  countryMap.getHoI4Tag((*sourceWorld->getProvince(std::stoi(strategy.getID())))->getOwner());
```

Comment:
Stoi can throw exceptions. Catch them and handle appropriately

Reply:
> Stoi can throw exceptions. Catch them and handle appropriately

Would this be good enough: `if (strategy.getType() == "conquer_prov" && std::stoi(strategy.getID()))`

Reply:
Unfortunately not. You actually need to stick the stoi part in a try/catch block, else if it goes wrong it'll stop the whole conversion with an unclear message. I'd separate it out entirely for simplicity:
```
int provinceToConquer;
try
{
  provinceToConquer = std::stoi(strategy.getID());
}
catch
{
  continue; // if errors should be recoverable and just skip that strategy
  throw std::exception("Bad AI strategy for " + country.first + ": " + strategy.getID()); // if its concerning enough to stop conversion to double-check the save
}